docs/embedded/serial-monitor.md

```diff
@@ -25,8 +25,9 @@ The Serial Monitor allows users to configure, monitor, and communicate with seri
 ## Capabilities
 
 - **Monitor a serial port**: Choose the **Start Monitoring** or **Stop Monitoring** button to control whether to monitor data coming from the port.
-- **Send data to a serial port**: Enter text into the text field at the bottom of the view. Use the **Enter** key or choose the **Send Message** button to send the data.
```

Comment:
Why did we remove the bit about using the "Enter" key?

Reply:
Oh I didn't realize we ever had that feature. I can add it back

Reply:
I just think it's useful to know for accessibility. Basically, if you're typing in the input box and hit enter, it should send.

Reply:
fair, I reverted it here. I'll update the code to allow enter too

Reply:
It should already be there, depending on your changes.